gradle.properties

```diff
@@ -1,8 +1,8 @@
 # Version of buf.build/bufbuild/protovalidate to use.
-protovalidate.version = v0.10.0
+protovalidate.version = v0.10.3
 
 # Arguments to the protovalidate-conformance CLI
-protovalidate.conformance.args = --strict --strict_message --strict_error
+protovalidate.conformance.args = --strict_message --strict_error --expected_failures=expected-failures.yaml
```

Comment:
The `--strict` flag was removed in https://github.com/bufbuild/protovalidate/pull/334